src/client/Client.cc

```diff
@@ -423,11 +423,11 @@ int Client::init()
 
   // logger
   PerfCountersBuilder plb(cct, "client", l_c_first, l_c_last);
-  plb.add_time_avg(l_c_reply, "reply");
-  plb.add_time_avg(l_c_lat, "lat");
-  plb.add_time_avg(l_c_wrlat, "wrlat");
-  plb.add_time_avg(l_c_owrlat, "owrlat");
-  plb.add_time_avg(l_c_ordlat, "ordlat");
+  plb.add_time_avg(l_c_reply, "reply", "Latency of receiving a reply on metadata request");
+  plb.add_time_avg(l_c_lat, "lat", "Latency of processing a metadata request");
+  plb.add_time_avg(l_c_wrlat, "wrlat", "Latency of a file data write operation");
+  plb.add_time_avg(l_c_owrlat, "owrlat", "Write to object latency");
+  plb.add_time_avg(l_c_ordlat, "ordlat", "Read from object latency");
```

Comment:
What I mean about the SyntheticClient is that that's just a test tool (and one we don't use), so these last two are basically dead values. You can't expect to use them to get any meaningful data; I'm actually inclined to remove them but if not the description shouldn't imply that they'll give you info on OSD access times for your ceph-fuse mount. :(